problemsets-mat223/linearalbegra.tex

```diff
@@ -1719,6 +1742,10 @@ \section*{Task 1.3: The Magic Carpet, Getting Back Home}
 				vectors. 
 			\end{solution}
 		\item Is $\Set{\vec u,\vec v,\vec w,\vec r}$ linearly independent?
+			\begin{solution}
+				No. We already know $\Set{\vec u,\vec v,\vec w}$ is not linearly
+				independent, so no set that contains it can be linearly independent.
```

Comment:
Be explicit from the def. "From before we know u+v-w=0 and therefore u+v-w+0r=0 is a non-trivial linear combination of ..." Students at this point will have the intuition about subsets but wouldn't be able to write down a proof.